projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() {
+	path, err := checkArg()
```

Comment:
This is a great example of somewhere that dependency injection could be used to make this function more testable - if this function took a `[]string` as an argument, rather than read `os.Args` itself, you could easily write tests for it.